### PR TITLE
Remove SCSS From Facepile Examples

### PR DESCRIPTION
**Pull request by pandasa123**
#### Pull request checklist

- [x] Addresses an existing issue: Related to #6075
- [x] Include a change request file using `$ yarn change`

#### Description of changes

- Removed SCSS file from examples 
- Followed `mergeStyleSets` pattern
- Updated snapshots
- Verified component changes visually

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11100)

### CODE REVIEW

**Comment by ecraig12345** [comment]
Looks like you need to update snapshots.

**Comment by pandasa123** [comment]
Sorry about that! Thank you so much @ecraig12345 for bearing with me

File: packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx [hunk 1]
```diff
@@ -5,14 +5,33 @@ import { Facepile, IFacepilePersona, IFacepileProps } from 'office-ui-fabric-rea
...
 
+const styles = mergeStyleSets({
+  container: {
+    maxWidth: 300
+  },
+  control: {
+    paddingTop: 20
+  },
+  slider: {
+    margin: '10px 0 10px 0'
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx]: >10px 0 10px 0 [](start = 13, length = 13)

Nit (can be ignored): Can be shortened to '10px 0'
Same below for dropdown

File: packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Overflow.Example.tsx [hunk 2]
```diff
@@ -3,7 +3,23 @@ import { IFacepileProps, Facepile, OverflowButtonType } from 'office-ui-fabric-r
...
+
+const styles = mergeStyleSets({
+  container: {
+    maxWidth: 300
+  },
+  control: {
+    paddingTop: 20
+  },
+  slider: {
+    margin: '10px 0 10px 0'
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Overflow.Example.tsx]: > '10px 0 10px 0' [](start = 11, length = 16)

Nit (can be ignored): Can be shortened to '10px 0'
Same below for dropdown

**Comment by maxwellred** [comment]
@pandasa123 I went ahead and made the changes in the PR.

**Comment by pandasa123** [comment]
@maxwellred Thank you so much! I saw the original and was thinking about that "nit" but wasn't sure about the style guide for this codebase. Do we have a centralised style document?

**Comment by maxwellred** [comment]
Hi @pandasa123, we do have [coding style guidelines](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Coding-Style) but nothing specific to this. However, we do prefer concise code as long as it's readable/understandable for others.

**Comment by pandasa123** [comment]
Ah I saw that one but got it. I'll keep that in mind for the future PRs!

**Comment by ecraig12345** [comment]
Those guidelines are pretty incomplete. There's also a ["typescript guidelines"](https://github.com/OfficeDev/office-ui-fabric-react/wiki/TypeScript-Guidelines) page (which is not actually specific to typescript) which has some additional information, and some duplicated information. (I'm working on de-duplicating those.) But we don't have any detailed guidelines for examples or for things like the margin string.

**Comment by ecraig12345** [comment]
They're now a bit more up to date but still don't cover examples or CSS.

**Pull request merged**